app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.java

```diff
@@ -415,4 +478,36 @@ public void onVoiceInputClick() {
             }
         }
     }
+
+    private String updateDescriptionInArticle(String articleText, String newDescription) {
```

Comment:
Would it be better if add `@NonNull` to the parameters?

Reply:
yep